### DISCUSSION TITLE
How is the back arrow supposed to work?

### DISCUSSION THREAD

**Discussion by sbug0** [elk-zone/elk#1676] [General]
If I for example open a toot by clicking on it, and I press the back arrow, it takes me back. (As I would expect).

But if I instead open a toot by clicking on it, and then click on the Home button, I'm taken to the Home feed, and the back arrow is still visible. (Not would I would expect).
Clicking the arrow takes me back to the old toot I was looking at.
Clicking the arrow again takes me back to the Home feed.

I would expect that clicking the Home button (or any other button in the list) would have cleared the back arrow.

**Comment by jcheroske** [comment]
Agreed. Also, I feel that if I click a toot and then reply to that toot, the back arrow should still take me back to the feed and not to the initial toot I clicked. Having to click the back arrow twice seems incorrect.
